### PR TITLE
update filter table  for server_processes_manager

### PR DESCRIPTION
**Pull request by kongtiaowang**

### CODE REVIEW

**Comment by driusan** [comment]
Code looks fine, but a third party should test it..

File: modules/server_processes_manager/jsx/columnFormatter.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+function formatColumn(column, cell, rowData) {
+    return <td>{cell}</td>;
```
Comment (driusan) [inline modules/server_processes_manager/jsx/columnFormatter.js]: Can you please add this to the Gruntfile.js too?

**Comment by kongtiaowang** [comment]
@driusan 
I have added the .js and .jsx into the Gruntfile.js, you could merge it.

**Comment by driusan** [comment]
This has a conflict

**Pull request merged**